blocked-edges/4.12.41-OVNKubeMasterDSPrestop.yaml

```diff
@@ -0,0 +1,21 @@
+to: 4.12.41
+from: .*
+url: https://issues.redhat.com/browse/SDN-4196
+name: OVNKubeMasterDSPrestop
+message: |-
+  Upgrades can get stuck on OVN clusters that were installed as 4.10 or earlier.
+matchingRules:
+- type: PromQL
+  promql:
+    promql:
+      topk(1,
+        label_replace(group(cluster_version{type="initial",version=~"4[.]([0-9]|10)[.].*"}),"born_by_4_10", "yes, born in 4.10 or earlier", "", "")
+        or
+        label_replace(0 * group(cluster_version{type="initial",version!~"4[.]([0-9]|10)[.].*"}),"born_by_4_10", "no, born in 4.11 or later", "", "")
+      )
+      * on () group_left (type)
```

Comment:
I think you want `resource` instead of `type` here to copy up the label from the `group by (resource) (...)` in the following block.

Reply:
I've pushed an update with this pivot as a part of aa3c8bfa -> cfe51de8.